### PR TITLE
TASK: Workspaces UX improvements

### PR DESCRIPTION
**Pull request by aertmann**
- Add review changes button to publish menu
- Styling improvements in workspaces new and edit forms
- Styling improvement to publish menu count badge
- Add missing jQuery dependency in TargetWorkspaceSelector
- Close target workspace selector when closing publish menu

### CODE REVIEW

**Comment by aertmann** [comment]
:+1:

**Comment by aertmann** [comment]
image
image
image

File: TYPO3.Neos/Resources/Private/Templates/Module/Management/Workspaces/Edit.html [hunk 1]
```diff
@@ -7,59 +7,61 @@
...
-					</f:then>
-					<f:else>
-						<f:form.select property="baseWorkspace" id="baseWorkspace" options="{baseWorkspaceOptions}" />
-					</f:else>
-				</f:if>
+			<div class="neos-control-group{f:validation.ifHasErrors(for: 'baseWorkspace', then: ' neos-error')}">
+				<label class="neos-control-label" for="baseWorkspace">{neos:backend.translate(id: 'workspaces.workspace.baseWorkspace', source: 'Modules', package: 'TYPO3.Neos')}</label>
+				<div class="neos-controls">
+					<f:if condition="{disableBaseWorkspaceSelector}">
+						<f:then>
```
Comment (dimaip) [inline TYPO3.Neos/Resources/Private/Templates/Module/Management/Workspaces/Edit.html]: Why extra indentation here?

Comment (aertmann) [inline TYPO3.Neos/Resources/Private/Templates/Module/Management/Workspaces/Edit.html]: this diff is rather useless, there's nothing changed except for indenting two spaces due to the added fieldset wrap tag

Comment (dimaip) [inline TYPO3.Neos/Resources/Private/Templates/Module/Management/Workspaces/Edit.html]: Oh gosh, that's weird, np then

**Comment by dimaip** [comment]
:+1: by reading. Do you want me to verify it?

**Comment by aertmann** [comment]
@dimaip: yes of course :)

**Comment by aertmann** [comment]
:+1: fixed two more issues

**Comment by robertlemke** [comment]
:+1: verified

The only improvement I could imagine is implementing some sort of return URI argument so that the "Back" button in the review view would bring you back to the content module. But that's for later and could be discussed.

**Comment by aertmann** [comment]
@robertlemke :+1: good idea

**Pull request merged**